java/server/src/org/openqa/selenium/server/RemoteControlConfiguration.java

```diff
@@ -121,6 +123,15 @@ public void setProfilesLocation(File profilesLocation) {
     this.profilesLocation = profilesLocation;
   }
 
+  public File getDumpScreenshots() {
+    return dumpScreenshots;
+  }
+
+  public void setDumpScreenshots(File dumpScreenshots) {
```

Comment:
Can we name it something like : getScreenshotLocation and setScreenshotLocation? 'dump' doesn't sound right with screenshots..